src/components/forms/PaymentOptionsForm/PaymentOptionsForm.tsx

```diff
@@ -0,0 +1,229 @@
+import { FundFrom, mARIOToken } from '@ar.io/sdk';
+import { Tooltip } from '@src/components/data-display';
+import { ArIOTokenIcon, TurboIcon } from '@src/components/icons';
+import { SelectDropdown } from '@src/components/inputs/Select';
+import {
+  useArIOLiquidBalance,
+  useArIOStakedAndVaultedBalance,
+} from '@src/hooks/useArIOBalance';
+import { formatARIOWithCommas } from '@src/utils';
+import { Circle, CircleCheck, CreditCard } from 'lucide-react';
+import { Tabs } from 'radix-ui';
+import { useMemo, useState } from 'react';
+
+export type PaymentMethod = 'card' | 'crypto' | 'credits';
+export type CryptoOptions = '$ARIO';
+
+const cryptoDropdownOptions: {
+  label: CryptoOptions;
+  value: CryptoOptions;
+}[] = [
+  {
+    label: '$ARIO',
```

Comment:
should use ticker from contract

Reply:
[2cdd28d](https://github.com/ar-io/arns-react/pull/731/commits/2cdd28d17b1b793fe4a2ada204146e39b4d0a166)